x-pack/plugins/enterprise_search/public/applications/app_search/components/synonyms/components/empty_state.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiPanel, EuiEmptyPrompt, EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { DOCS_PREFIX } from '../../../routes';
+
+import { SynonymIcon } from './';
+
+export const EmptyState: React.FC = () => {
+  return (
+    <EuiPanel color="subdued">
+      <EuiEmptyPrompt
+        iconType={SynonymIcon}
+        title={
+          <h2>
+            {i18n.translate('xpack.enterpriseSearch.appSearch.engine.synonyms.empty.title', {
+              defaultMessage: 'Create your first synonym set',
+            })}
+          </h2>
+        }
+        body={
+          <p>
+            {i18n.translate('xpack.enterpriseSearch.appSearch.engine.synonyms.empty.description', {
+              defaultMessage:
+                'Synonyms relate queries with similar context or meaning together. Use them to guide users to relevant content.',
```

Comment:
I made this up to match Davey's new empty state polish/pass (https://github.com/elastic/kibana/pull/97875). Feel free to give copy suggestions!!